firebase-perf/src/main/java/com/google/firebase/perf/metrics/FrameMetricsCalculator.java

```diff
@@ -49,6 +49,13 @@ public int getSlowFrames() {
     public int getTotalFrames() {
       return totalFrames;
     }
+
+    public PerfFrameMetrics subtract(PerfFrameMetrics b) {
```

Comment:
I think need to get a better name for this. "Subtract" does not sound any intuitive here. Some recommendation could be `differenceInFrameMetricsFromSnapshot(PerfFrameMetrics oldSnapshot)`.

Reply:
I thought "subtract" because we are literally subtracting the counts for total/slow/frozen frames. Also snapshot is a concept that should be contained in `FrameMetricsRecorder` imo. `PerfFrameMetrics` shouldn't know anything about snapshot, and its method naming should be whatever this operation does from the perspective of `PerfFrameMetrics`.

Reply:
Can you at least add a javadoc to explain what this method does?

Personally, I don't know why we would be subtracting some metrics from others (as in, what is the use-case?) :)